### ISSUE TITLE
EsLint Errors Prevent backend in vue from compiling

### ISSUE THREAD

**Issue by adukugideon** [chrisleekr/nodejs-vuejs-mysql-boilerplate#67]
I get this specific error `[eslint] Failed to load plugin 'vuejs-accessibility' declared in '.eslintrc.js » @vue/eslint-config-airbnb': Cannot find module 'eslint-plugin-vuejs-accessibility`

**Comment by LenonTolfo** [comment]
I am getting the same error, did you manage to find a solution ?
